spec/models/api/base_spec.rb

```diff
@@ -6,6 +6,21 @@
     allow(Rails.application.config.http_client_xi).to receive(:retrieve).and_call_original
   end
 
+  describe '#initialize' do
+    context 'when initialized with unknown attributes' do
```

Comment:
Validates that we can instantiate a model without knowing about the attributes explicitly